repo/packages/H/hdfs/5/config.json

```diff
@@ -0,0 +1,357 @@
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"id": "http://hdfs/dcos/mesosphere.com",
+	"type": "object",
+	"properties": {
+		"service": {
+			"id": "http://hdfs/dcos/mesosphere.com/service",
+			"type": "object",
+			"description": "DC/OS service configuration properties",
+			"properties": {
+				"name": {
+					"id": "http://hdfs/dcos/mesosphere.com/service/name",
+					"type": "string",
+					"description": "The name of the HDFS service instance.",
+					"default": "hdfs"
+				},
+				"role": {
+					"id": "http://hdfs/dcos/mesosphere.com/service/role",
+					"type": "string",
+					"description": "The role of the HDFS service instance.",
+					"default": "hdfs"
+				},
+				"principal": {
+					"id": "http://hdfs/dcos/mesosphere.com/service/principal",
+					"type": "string",
+					"description": "The principal for the HDFS service instance.",
+					"default": "hdfs"
+				},
+				"cpus": {
+					"id": "http://hdfs/dcos/mesosphere.com/service/cpus",
+					"type": "number",
+					"description": "The cpu shares used by the HDFS service scheduler.",
+					"default": 0.5
+				},
+				"mem": {
+					"id": "http://hdfs/dcos/mesosphere.com/service/mem",
+					"type": "integer",
+					"description": "The memory used by the HDFS service scheduler in Mb.",
+					"default": 2048
+				},
+				"heap": {
+					"id": "http://hdfs/dcos/mesosphere.com/service/heap",
+					"type": "integer",
+					"description": "The JVM heap used by the HDFS service scheduler in Mb.",
+					"default": 1024
+				},
+				"api_port": {
+					"id": "http://hdfs/dcos/mesosphere.com/service/api_port",
+					"type": "integer",
+					"description": "The port that the HDFS service API will accept on.",
+					"default": 9000
+				},
+				"placement_strategy": {
+					"id": "http://hdfs/dcos/mesosphere.com/service/placement_strategy",
+					"type": "string",
+					"description": "The placement strategy for HDFS nodes. This should always be NODE.",
+					"default": "NODE"
+				},
+				"secret_name": {
+					"type": "string",
+					"description": "Name of the authentication Secret to use for DC/OS service authentication. This should be left empty unless service authentication is needed.",
```

Comment:
Should probably mention that this is the name of the secret in the secret store, not a local file or some other name